ui/src/app/job-list/table/table.component.ts

```diff
@@ -140,17 +140,33 @@ export class JobsTableComponent implements OnInit {
   onAbortJobs(): void {
     const aborts: Promise<void>[] = [];
     const selected = this.selection.selected.slice();
+
+    let numErrs = 0;
+    const ref = this.errorBar.open(
```

Comment:
nit: maybe rename from error bar since its not only for errors now?

Reply:
Good idea, done.